eks/cluster-loader/configs/cluster-autoscaler/config.yaml

```diff
@@ -0,0 +1,132 @@
+# Constants
+# Currently set to 1 namespace.
+{{$NAMESPACES := DefaultParam .namespaces 1}}
+{{$PODS_PER_NODE := DefaultParam .podsPerNode 10}}
+{{$QPS := DefaultParam .qps 10}}
+{{$POD_RESOURCE_REQUEST_FUZZ_FACTOR := DefaultParam .fuzzFactor .9}}
+{{$POD_GROUP_LABEL := DefaultParam .groupLabel "cluster-loader"}}
```

Comment:
Let's be consistent with these variables. How about

{{$VariableName := DefaultParam .VariableName "default-value }}

Reply:
+1